src/test/components/ButtonWithPanel.test.js

```diff
@@ -21,25 +23,15 @@ describe('shared/ButtonWithPanel', function() {
         }
       />
     );
-    expect(container.firstChild).toMatchSnapshot();
+  }
+  afterEach(cleanup);
```

Comment:
we don't need this cleanup anymore, because it's been added in https://github.com/devtools-html/perf.html/blob/master/src/test/setup.js#L18. Can you please remove it? (and you can remove the import above too)